src/Analyzers/Core/CodeFixes/Formatting/FormattingCodeFixProvider.cs

```diff
@@ -28,12 +26,14 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             foreach (var diagnostic in context.Diagnostics)
             {
+#pragma warning disable RS0005 // Do not use generic 'CodeAction.Create' to create 'CodeAction'
                 context.RegisterCodeFix(
```

Comment:
@mavasani The one that used to live in `Features` was inheriting `SyntaxEditorBasedCodeFixProvider`. While the one that lived in `CodeStyle` was inheriting `CodeFixProvider` directly. Does it matter what do I inherit when merging both in `Analyzers` layer?

Reply:
Moving to `SyntaxEditorBasedCodeFixProvider` is preferable if possible, and you don't see any breaking changes.

Reply:
@mavasani Done in last commit (5e4b835)